web/docs/data-model/operations/actions.md

```diff
@@ -182,27 +186,62 @@ export const markTaskAsDone: MarkTaskAsDone<Pick<Task, 'id'>, void> = (
 }
 ```
 
+<SuperjsonNote />
+
+#### Type support for Actions
+
 Wasp automatically generates the types `CreateTask` and `MarkTaskAsDone` based on the declarations in your Wasp file:
 
 - `CreateTask` is a generic type that Wasp automatically generated based on the Action declaration for `createTask`.
 - `MarkTaskAsDone` is a generic type that Wasp automatically generated based on the Action declaration for `markTaskAsDone`.
 
-You can use these types to specify the Action's input and output types.
+Use these types to type the Action's implementation.
+Annotating Actions is optional but very helpful.
+Doing so properly types the Action's context.
```

Comment:
Can we maybe make this into one or two more complex sentences, it feels a bit like a draft text waiting to be shaped.